auslib/test/test_db.py

```diff
@@ -1892,6 +1892,16 @@ def setUp(self):
                                       product="foo", channel="foo*", data_version=1)
         self.paths.t.insert().execute(rule_id=10, priority=100, buildTarget="g", mapping="g", fallbackMapping='fallback', backgroundRate=100,
                                       update_type="z", product="foo", channel="foo", data_version=1)
+
+        self.paths.t.insert().execute(rule_id=11, priority=100, buildTarget='h', mapping='h', systemCapabilities='GenuineIntel,SSE2', update_type='z',
+                                      product='a', channel='a', data_version=1)
+        self.paths.t.insert().execute(rule_id=12, priority=100, buildTarget='i', mapping='i', systemCapabilities='GenuineIntel&&SSE2', update_type='z',
+                                      product='a', channel='a', data_version=1, version='5.0')
+        self.paths.t.insert().execute(rule_id=13, priority=100, buildTarget='j', mapping='j', systemCapabilities='!SSE3', update_type='z', product='a',
+                                      channel='a', data_version=1)
+        self.paths.t.insert().execute(rule_id=14, priority=100, buildTarget='k', mapping='k', systemCapabilities='!GenuineIntel&&SSE3,SSE2', update_type='z',
```

Comment:
Could we support whitespace (even if it's just a single space) around the operators? It would greatly improve readability IMO.